optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/MeetingScheduling/MeetingScheduling-section.adoc

```diff
@@ -21,7 +23,11 @@ Medium constraints:
 Soft constraints:
 
 * Sooner rather than later: Schedule all meetings as soon as possible.
-
+* A break between meetings: Any two meetings should have at least one time grain break between them.
+* Overlapping meetings: To minimize the number of meetings in parallel so people don't have to choose one meeting over the other.
+* Assign larger rooms first: If a larger room is available any meeting should be assigned to that room in order to accommodate as many people as possible
+    even if they haven't signed up to that meeting.
+* Room stability: If a person has two consecutive meetings with two or less time grains break between them they better be in the same room.
 
```

Comment:
code style suggestion: for clarity in the docs, we prefer 2 empty lines above a section header (so don't remove the empty line)